ee/modules/007-registrypackages/images/kubernetes-cni-alteros/werf.inc.yaml

```diff
@@ -37,4 +38,5 @@ shell:
   setup:
   - RPM_PACKAGE="$(curl -s https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64/repodata/primary.xml | grep "<location href=" | grep "kubernetes-cni-{{ $version }}-0" | awk -F "\"" '{print $2}')"
   - curl -sfL https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64/${RPM_PACKAGE} --output /kubernetes-cni.x86_64.rpm
+  - curl -sfL https://github.com/flannel-io/cni-plugin/releases/download/v1.1.2/cni-plugin-flannel-linux-amd64-v1.1.2.tgz | tar -xz ./flannel-amd64 && mv flannel-amd64 flannel
```

Comment:
How can it work in air-gapped environments?

Reply:
It's a build phase

Reply:
What I was thinking, IDK 😅